src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -258,7 +258,7 @@ public enum NuGetLogCode
         NU1603 = 1603,
 
         /// <summary>
-        /// Project dependency does not include a lower bound.
+        /// Project dependency has no resolved version or does not include a lower bound.
```

Comment:
this statement not right. 
NU1604 is about what's declared, not about what got resolved.

Reply:
Ok. Reverted this change and created new NU1609 warning.